src/components/schedule-visualizer/door/door.tsx

```diff
@@ -0,0 +1,117 @@
+import * as RomiCore from '@osrf/romi-js-core-interfaces';
+import React from 'react';
+import { Spinner } from '../spinner';
+import SingleHingeDoor from './door-single-hinge';
+import DoubleHingeDoor from './door-double-hinge';
+import SingleSlideDoor from './door-single-slide';
+import DoubleSlideDoor from './door-double-slide';
+import DefaultDoor from './door-default';
+
+export enum DoorMode {
+  CLOSE,
+  PROCESS,
+  OPEN,
+}
+
+export enum DoorType {
```

Comment:
Is it possible to reuse the constants in `romi-js-core-interfaces`?

Reply:
Done! Thanks!